libs/src/services/ethContracts/governanceClient.js

```diff
@@ -174,12 +158,26 @@ class GovernanceClient extends ContractClient {
     let events = await contract.getPastEvents('ProposalSubmitted', {
       fromBlock: queryStartBlock,
       filter: {
-        proposer: addresses
+        _proposer: addresses
       }
     })
     return events.map(this.formatProposalEvent)
   }
 
+  async getProposalSubmission (
+    proposalId,
+    queryStartBlock = 0
+  ) {
+    const contract = await this.getContract()
+    const events = await contract.getPastEvents('ProposalSubmitted', {
```

Comment:
i think this should always be length 1 right? like a proposal can never be submitted and have the same ID again

Reply:
yeah that's why i just willy nillly did events[0]. probably could do something else, but it works!